developer-guides/recommended-setup.mdx

```diff
@@ -0,0 +1,11 @@
+---
+title: "Recommended setup"
+description: "A developer's introduction to using Koala's client-side setup"
+---
+
+Outline
+
+- why we recommend client-side
+- pageviews
+- session time
```

Comment:
I probably would talk about "autotracking" and then go into pageviews, session time and form submissions.

👍